Update README.md - Adding SBC Linux updating process.

Will-wastelander opened the pull request:
Added SBC Linux update process.

<!--
Thank you for your contribution to our project! Please fill out the following template to help reviewers understand your changes.
-->

## Description
<!--
-->
Providing a way to update WebUI on SBC Linux systems like FemtoFox carrier board for LuckyFox. This allows for users to have an updated WebUI.

## Changes Made
<!--
-->
- Added section `SBC Linux`

## Testing Done
<!--
-->
Ran commands via SSH to FemtoFox carrier board for LuckyFox

## Checklist
<!--
Check all that apply. If an item doesn't apply to your PR, you can leave it unchecked or remove it.
-->
- [ ] Code follows project style guidelines
- [X] Documentation has been updated or added
- [ ] Tests have been added or updated
- [ ] All CI checks pass
- [ ] Dependent changes have been merged

## Additional Notes
<!--
-->
This process may also work with Raspberry Pi, and other SBC systems. I do not have another carrier board that runs Linux at this time.

Will-wastelander committed:
Update README.md

Added SBC Linux update process.

danditomaso commented on README.md:
````diff
@@ -50,6 +50,28 @@ docker run -d -p 8080:8080 --restart always --name Meshtastic-Web ghcr.io/meshta
 podman run -d -p 8080:8080 --restart always --name Meshtastic-Web ghcr.io/meshtastic/web
 ```
 
+## SBC Linux
````
I'm wondering if we should include "MeshtasticD" in the title?

Hunter275 replied:
Also, can just be "Linux" as it doesn't have to be on an SBC

danditomaso commented on README.md:
````diff
@@ -50,6 +50,28 @@ docker run -d -p 8080:8080 --restart always --name Meshtastic-Web ghcr.io/meshta
+
+Download latest Build.tar file, extract to `/usr/shared/meshtasticd/web`. 
+
+```bash
````
I think we should explictly tell them to change dir's to their home directory, so they can do that work in that folder prior to moving the web folder into meshtasticd.

Will-wastelander replied:
This was the implied location. This line was more of a description of what the overall commands/actions below do.

danditomaso reviewed (commented):
Overall looks good, left a couple of comments with some suggestions.

Will-wastelander committed:
Update README.md

Added command to make sure user is in home directory before completing any commands.
